misk-jobqueue/src/main/kotlin/misk/jobqueue/JobConsumer.kt

```diff
@@ -7,11 +7,11 @@ interface JobConsumer {
    * begin receiving messages from the underlying job queue and dispatch them to the provided
    * handler. A service may only have one subscription outstanding per queue
    */
-  fun subscribe(queueName: QueueName, handler: JobHandler)
+  fun subscribe(queueNames: List<QueueName>, handler: JobHandler)
```

Comment:
again not backwards compatible

Reply:
the `subscribe` extension method handles this case.